pybedlite/overlap_detector.py

```diff
@@ -125,6 +144,62 @@ def from_bedrecord(cls: Type["Interval"], record: BedRecord) -> "Interval":
             name=record.name,
         )
 
+    @classmethod
+    def from_ucsc(
+        cls: Type["Interval"],
+        ucsc: str,
```

Comment:
I don't love this as a parameter name, but I was concerned that using `interval` (the most sensible alternative name I could think of) would be confusing since it's a `str` and not an `Interval`. (And I don't prefer exposing the type in a variable/parameter name e.g. `interval_str` unless it's only used internally)

Reply:
one could just say `value`?